recipes-bsp/rpi-boot-setting/files/boot.cmd

```diff
@@ -1,4 +1,9 @@
-fatload mmc 0 ${kernel_addr_r} Image
-fatload mmc 0 ${fdt_addr_r} bcm2837-rpi-3-b-plus.dtb
-setenv bootargs dwc_otg.lpm_enable=0 earlyprintk root=/dev/mmcblk0p2 rootfstype=ext4 rootwait
-booti ${kernel_addr_r} - ${fdt_addr_r}
+setenv setbootargs 'setenv bootargs dwc_otg.lpm_enable=0 earlyprintk root=PARTUUID=${uuid} rootfstype=ext4 rootwait'
+setenv setuuid 'part uuid mmc ${devnum}:2 uuid'
+setenv bootlinux 'run setuuid && run setbootargs && fatload mmc ${devnum} ${kernel_addr_r} Image && fatload mmc ${devnum} ${fdt_addr_r} ${fdt_name} && booti ${kernel_addr_r} - ${fdt_addr_r}'
+
+setenv boot_rpi3 'setenv fdt_name bcm2837-rpi-3-b-plus.dtb && setenv devnum 0 && run bootlinux'
+setenv boot_rpi4 'setenv fdt_name bcm2711-rpi-4-b.dtb && setenv devnum 1 && run bootlinux'
+
+run boot_rpi3
+run boot_rpi4
```

Comment:
Don't mix up boot sequences for 4B and 3B+ in a single boot.cmd file. Create another boot.cmd file only for 4B.

Also, if you want to change the boot sequence for 3B+, separate only the change for modifying the boot sequence for 3B+ into another commit, adding the reason as a commit message.

The commit order as below would be preferred.

1. Change the boot sequence for 3B+.
2. Add initial support for Raspberry Pi 4 Model B
3. doc: raspberrypi: Add documentation to build an image for Raspberry Pi 4B

Otherwise, don't change the boot sequence for 3B+.